roles/pulp_webserver/tasks/apache.yml

```diff
@@ -22,9 +22,20 @@
           dest: /etc/apache2/sites-enabled/pulp-vhost.conf
         - src: ../mods-available/proxy.load
           dest: /etc/apache2/mods-enabled/proxy.load
+        - src: ../mods-available/proxy_http.load
+          dest: /etc/apache2/mods-enabled/proxy_http.load
+        - src: ../mods-available/ssl.load
+          dest: /etc/apache2/mods-enabled/ssl.load
+        - src: ../mods-available/headers.load
+          dest: /etc/apache2/mods-enabled/headers.load
```

Comment:
Is this necessary to make the tests pass?

It deserves a changelog entry (and a separate commit.)

Reply:
On debian10 box apache won't start without these.

Reply:
can we use apache2_module here?

https://docs.ansible.com/ansible/latest/collections/community/general/apache2_module_module.html